HW04/ekaterina_balabanovich/lib/work_state/work_state.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+class WorkState
+  def process(members, homework); end
+end
```

Comment:
I would prefer to call it `Base` or `Abstract` to represent a real value of this class
```suggestion
module WorkState
  class Base
    def process(members, homework); end
  end
end
```